app/support/template.hbs

```diff
@@ -116,7 +116,7 @@
                 >
                     {{t 'support.prereg_link_text'}}
                 </OsfLink>
-                {{t 'support.prereg_paragraph_2'}}
+                {{t 'general.period'}}
```

Comment:
In order to not have extra space between the last word and the period, this needs to be:
```suggestion
                {{~t 'general.period'}}
```